test/gtest/uct/ib/test_rc.cc

```diff
@@ -331,15 +331,21 @@ uct_rc_mlx5_iface_common_t* test_rc_mp_xrq::rc_mlx5_iface(entity &e)
     return ucs_derived_of(e.iface(), uct_rc_mlx5_iface_common_t);
 }
 
-void test_rc_mp_xrq::init()
+void test_rc_mp_xrq::set_env_var_or_skip(void *config, const char *var,
+                                         const char *val)
 {
-    ucs_status_t status1 = uct_config_modify(m_iface_config,
-                                             "RC_TM_MP_NUM_STRIDES", "8");
-    ucs_status_t status2 = uct_config_modify(m_iface_config,
-                                             "RC_TM_ENABLE", "y");
-    if ((status1 != UCS_OK) || (status2 != UCS_OK)) {
-        UCS_TEST_SKIP_R("No MP XRQ support");
+    ucs_status_t status = uct_config_modify(config, var, val);
+    if (status != UCS_OK) {
+        ucs_warn("%s", ucs_status_string(status));
```

Comment:
[q] the test cases will be marked as failed, right? since it will catch that number of warning > 0
it's ok?

Reply:
leftover..